joint_limits:
  joint_urs100:
    has_position_limits: true
    min_position: -2.87979
    max_position: 2.87979
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0